crates/containerd-shim-wasm/src/sandbox/path.rs

```diff
@@ -1,7 +1,7 @@
 use std::path::{Path, PathBuf};
 
 /// PathResolve allows to resolve a file path in a set of directories.
-pub trait PathResolve {
+pub(crate) trait PathResolve {
```

Comment:
this is a breaking change I think? Can you document it in the CHANGELOG?

Reply:
It is not. This trait was not accessible because the module was not public and it was not reexported anywhere.